package.json

```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/switchfly/ember-cli-mocha",
   "dependencies": {
-    "broccoli-jshint": "^0.5.3"
+    "broccoli-jshint": "^0.5.6"
```

Comment:
We need *at least* this version for JSHint to not complain about ES6 template literals.